package.json

```diff
@@ -736,8 +736,10 @@
     "@kbn/search-response-warnings": "link:packages/kbn-search-response-warnings",
     "@kbn/search-types": "link:packages/kbn-search-types",
     "@kbn/searchprofiler-plugin": "link:x-pack/plugins/searchprofiler",
+    "@kbn/security-api-key-management": "link:x-pack/packages/security/api_key_management",
     "@kbn/security-hardening": "link:packages/kbn-security-hardening",
     "@kbn/security-plugin": "link:x-pack/plugins/security",
+    "@kbn/security-plugin-form-components": "link:x-pack/packages/security/plugin_form_components",
```

Comment:
nit: could we rename the folder to be `packages/security/form_components` and `packages/security/api_key_creation_components` or something similar? The package itself isn't handling all of API key management so it'd be clearer for other users of the package.

Reply:
The reason I went with api_key_management is that it also includes a bunch of types and some functions that are more generic, and there's update functionality as well. It doesn't include all of the API key management stuff, that's true, but I don't think `api_key_creation_components` covers what's happening either.

I'll rename the form_components plugin :)

Reply:
Renamed form_components in https://github.com/elastic/kibana/pull/186171/commits/817aab70138fa7f803c08a3cbf57137f4a5ea81e